controller/webhook/server.go

```diff
@@ -79,6 +119,31 @@ func (s *Server) Start() {
 	}
 }
 
+// getCertificate returns a function that provides the TLS server with the current cert
+func (s *Server) getCertificate() func(*tls.ClientHelloInfo) (*tls.Certificate, error) {
```

Comment:
it is possible to not return an anonymous function here and instead just use this function directly?  i.e. set `server.TSLConfig.GetCertificate = s.getCertificate`

Reply:
Yep, that does the trick as well :+1: